docs/tutorials/deployment.md

```diff
@@ -6,64 +6,74 @@ A few basic concepts about this process:
 __"Export method"__ is how a Python model is fully serialized to a deployable format.
 We support the following export methods:
 
-* `tracing`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it
-* `scripting`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it
+* `tracing`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it\
+This method produces a standardized ONNX graph that can be used by any ONNX runtime, such as \
+[ONNX Runtime](https://onnxruntime.ai/)
+* `scripting`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it\
+This method produces a standardized ONNX graph that can be used by any ONNX runtime, such as \
+[ONNX Runtime](https://onnxruntime.ai/)
 * `caffe2_tracing`: replace parts of the model by caffe2 operators, then use tracing.
+This method produces a Caffe2-specific ONNX graph that can be used only by the [Caffe2](https://caffe2.ai/).
+Caffe2 support has been ***DEPRECATED*** by PyTorch, therefore, we don't plan to support it, but contributions are welcome.
```

Comment:
Might be a good idea to delete everything about caffe2 from this doc to simplify things.

Reply:
I am OK in removing any reference to Caffe2 ONNX export, but my previous PR already did the bulk of the work to make it work.

Last I heard, Meta still uses Caffe2 internally, so the removal might break their thing

Reply:
I'm only suggesting to remove docs since probably no one externally is using them. But either way is fine.